amundsen_application/api/utils/metadata_utils.py

```diff
@@ -149,9 +148,9 @@ def marshall_dashboard_partial(dashboard_dict: Dict) -> Dict:
     :param dashboard_dict: Dict of partial dashboard metadata
     :return: partial dashboard Dict
     """
-    schema = DashboardSummarySchema(strict=True)
-    dashboard: DashboardSummary = schema.load(dashboard_dict).data
-    results = schema.dump(dashboard).data
+    schema = DashboardSummarySchema(unknown=EXCLUDE)
```

Comment:
a test contains `total_usage` which is not on the schema. Should the schema (which is in `common`) be updated?

Reply:
it doesn't look like that field ever existed in common moel in the first place, weird

Reply:
maybe the model is outdated, cc @jinhyukchang in case Jin remembered